src/genophenocorr/preprocessing/_vep.py

```diff
@@ -8,6 +8,7 @@
 from genophenocorr.model.genome import Region
 from ._api import FunctionalAnnotator, ProteinMetadataService
 
+SPLICE_EFFECTS = [VariantEffect.SPLICE_ACCEPTOR_VARIANT, VariantEffect.SPLICE_DONOR_VARIANT, VariantEffect.SPLICE_DONOR_5TH_BASE_VARIANT, VariantEffect.SPLICE_POLYPYRIMIDINE_TRACT_VARIANT]
```

Comment:
As a rule of thumb, you should always try to keep a variable declaration as close to the usage as possible.

In this case, the best place to put the `SPLICE_EFFECTS` is likely a class field, because it is a static variable.

Please consider moving `SPLICE_EFFECTS` closer to usage.